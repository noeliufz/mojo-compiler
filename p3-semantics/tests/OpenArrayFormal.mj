proc foo(a: array of int) {
}
