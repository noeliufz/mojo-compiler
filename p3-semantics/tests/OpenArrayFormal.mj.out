BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool={false:bool=0, true:bool=1}
_BUILTIN.false:bool=0
_BUILTIN.true:bool=1
_BUILTIN.byte=[-128..127]
_BUILTIN.char={}
_BUILTIN.Null=ref null
_BUILTIN.null:Null=0
_BUILTIN.Refany=ref null
_BUILTIN.Address=ref null
_BUILTIN.Root=null object {}
_BUILTIN.Text=ref null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.Ord:(...)
_BUILTIN.Val:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
_BUILTIN.Loophole:(...)
BEGIN OpenArrayFormal
OpenArrayFormal.foo:(VALUE a:array null of int)
BEGIN OpenArrayFormal.foo
OpenArrayFormal.foo.a:array null of int
END OpenArrayFormal.foo
END OpenArrayFormal
END _BUILTIN
